shenyu-plugin/shenyu-plugin-brpc/src/main/java/org/apache/shenyu/plugin/brpc/cache/ApplicationConfigCache.java

```diff
@@ -116,9 +139,7 @@ public AsyncGenericService build(final MetaData metaData) {
         if (Objects.isNull(clientConfig)) {
             throw new UnsupportedOperationException("unsupport!!");
         }
```

Comment:
When `org.apache.shenyu.plugin.brpc.cache.ApplicationConfigCache#init` is not called, try to initialize `org.apache.shenyu.plugin.brpc.cache.ApplicationConfigCache#initService` , `org.apache.shenyu.plugin.brpc.cache.ApplicationConfigCache#build ` will throw `UnsupportedOperationException`.

Therefore, we should delay the action of building `AsyncGenericService` until the `runtime`. Of course, caching `AsyncGenericService`  is recommended.